libra/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: "3"
+services:
+
+  mongodb:
+    container_name: libra_mongodb
+    image: mongo:4.0
+    environment:
+       - MONGO_DATA_DIR=/data/db
+       - MONGO_LOG_DIR=/dev/null
+    volumes:
+       - mongodbdata:/data/db 
+    ports:
+      - "127.0.0.1:${ENV_MONGO_PORT:-27017}:27017"
+    command: mongod 
+    networks:
+      local:
+        aliases:
+          - mongodb
+
+  spring:
+    container_name: libra_spring
+    image: kingstonduo/kpmp-spring
+    ports:
+      - "${ENV_APACHE_TOMCAT_PORT:-3030}:3030"
```

Comment:
Don't we want this to be 127.0.0.1:{port} so that it's only exposed to the localhost and not the outside world?

Reply:
Based on Becky Reamy's feedback, it sounds like we will leave this in for development for now.  She has a note to write a card to address this prior to release.